README.md

```diff
@@ -1 +1,62 @@
-# Awesome-books-with-ES6
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# awesome-books-project-modules
+
+
+![](file:///home/moise/my-awesome-books-modules/index.html)
+
+# Awesome books: plain JavaScript with objects
+
+> This my awesome books project using modules to separate JavaScript files into small chunks. 
+
+![screenshot](awesome-book-img/awesome-book.png).
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript ES6/Modules
+- Visual Studio Code.
+- Linters
+
+## Live Demo
+
+[github page](https://moise-mulungu.github.io/Awesome-books-with-ES6/)
+
+[live demo](http://127.0.0.1:5500/index.html)
+
+## Getting Started
+
+To get a local copy of this app you can use :
+```
+git clone  https://github.com/moise-mulungu/Awesome-books-with-ES6/tree/create-es6-branch
+```
+To make how it works, use live-server from your local.
+
+in this app, you can open three different pages, with an update time which runs perfectly, when you add a book it automatically adds itself on the list.
```

Comment:
- [ ] In order for your luxon package to work users need to run `npm install` to install packages. Kindly add this command as necessary to run the app locally.